### PR TITLE
Add Korean version "Getting Started"

### PR DESCRIPTION
**Pull request by JungGaBin**
*Add Korean index.html.

*Add Korean main.css.

*Add Korean screenshots.

*Add Korean Getting Started url.

### CODE REVIEW

File: samples/ko/Getting Started/index.html
Comment (ficristo) [inline samples/ko/Getting Started/index.html]: I don't know korean language, is it possible to add a space after BRACKETS? Here and below.

File: samples/ko/Getting Started/index.html
Comment (ficristo) [inline samples/ko/Getting Started/index.html]: The string inside the `content` attribute can be translated.

File: samples/ko/Getting Started/index.html
Comment (ficristo) [inline samples/ko/Getting Started/index.html]: Here and in the other places you should close the html tags correctly: no whitespace after `</`

File: samples/ko/Getting Started/index.html
Comment (ficristo) [inline samples/ko/Getting Started/index.html]: You should match the comments as in the root translation:
so `<!-- <samp> -->` and not `<! - <samp> ->`
Please check all comments

File: samples/ko/Getting Started/index.html
Comment (ficristo) [inline samples/ko/Getting Started/index.html]: The string inside the `alt` attribute can be translated.

File: samples/ko/Getting Started/index.html
Comment (ficristo) [inline samples/ko/Getting Started/index.html]: If you want you can translate also this sentence.

File: samples/ko/Getting Started/index.html
Comment (ficristo) [inline samples/ko/Getting Started/index.html]: If you want you can translate this comments.

**Review by ficristo — COMMENTED** [review]
Overall it seems good.
@kimohyeong @JeongGoEun can you help to check the translations?

**Comment by JungGaBin** [comment]
@ficristo 

I checked your comment.  i got it.
i will fix it , right now.

File: samples/ko/Getting Started/index.html
Comment (JungGaBin) [reply samples/ko/Getting Started/index.html]: @ficristo

Comment (ficristo) [reply samples/ko/Getting Started/index.html]: Yes. I don't know if there is a grammar rule that forbid it, but otherwise I think it would look nicer with a whitespace.

Comment (JungGaBin) [reply samples/ko/Getting Started/index.html]: @ficristo  Okay, I will fix it.

Comment (JungGaBin) [reply samples/ko/Getting Started/index.html]: @ficristo 

I fixed all your comments.
Please confirm.

**Comment by ficristo** [comment]
Please do not open new PR. Just push your changes here.

**Comment by JungGaBin** [comment]
@ficristo  I got it.

**Comment by ficristo** [comment]
You should move the screenshot in a `screenshots` folder as before.
I didn't notice in my first pass, but the screenshot should contain the quick edit window, as the root one.

**Comment by JungGaBin** [comment]
@ficristo  I fixed it.

**Comment by JungGaBin** [comment]
@ficristo 
I fixed it finally.
Plz confirm that. Thank u

**Review by ficristo — APPROVED** [review]
LGTM.
Lets wait a couple of days to see if there are comments from other users. After that a committer will merge it.
Thank you for your contribution.

**Comment by JeongGoEun** [comment]
Hi @ficristo I'm Korean :)
All translated sentences were correctly translated.
This document is likely to be of use to Koreans using Brackets.
Thanks!

**Comment by Jeonghyunsu** [comment]
It is good translation.

**Comment by ficristo** [comment]
@JungGaBin @JeongGoEun @Jeonghyunsu thank you.

**Pull request merged**
